src/libs/ReportUtils.js

```diff
@@ -1539,14 +1550,25 @@ function canEditReportAction(reportAction) {
 /**
  * Gets all transactions on an IOU report with a receipt
  *
- * @param {Object|null} iouReportID
+ * @param {string|null} iouReportID
  * @returns {[Object]}
  */
 function getTransactionsWithReceipts(iouReportID) {
     const allTransactions = TransactionUtils.getAllReportTransactions(iouReportID);
     return _.filter(allTransactions, (transaction) => TransactionUtils.hasReceipt(transaction));
 }
 
+/**
+ * Gets all transactions of the IOU report that are of the Distance Request type
+ *
+ * @param {string|null} iouReportID
+ * @returns {[Object]}
+ */
+function getDistanceRequestTransactions(iouReportID) {
```

Comment:
I think we don't need this method as we are only calling it once and that too only to check if we have only distance requests here:
```
const distanceRequestTransactions = ReportUtils.getDistanceRequestTransactions(props.iouReportID);
    const hasOnlyDistanceRequests = numberOfRequests === _.size(distanceRequestTransactions);
```